include/ignition/math/Ellipsoid.hh

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef IGNITION_MATH_ELLIPSOID_HH_
+#define IGNITION_MATH_ELLIPSOID_HH_
+
+#include <optional>
+#include "ignition/math/MassMatrix3.hh"
+#include "ignition/math/Material.hh"
+
+namespace ignition
+{
+  namespace math
+  {
+    // Foward declarations
+    class EllipsoidPrivate;
```

Comment:
Did you mean to create this class? I see the same thing sneaked into the `Capsule` class.

Reply:
I'll take the liberty of removing this so it can rerun the windows job

Reply:
there is no `CapsulePrivate`, so that forward declaration could be removed as well

Reply:
Thanks @scpeters. We can blame my gratuitous use of copy/paste.